BoxContentSDK/BoxContentSDK/BOXContentSDKErrors.h

```diff
@@ -77,6 +77,12 @@ typedef NS_ENUM(NSUInteger, BOXContentSDKStreamError) {
     BOXContentSDKStreamErrorReadFailed = 30001
 };
 
+typedef NS_ENUM(NSUInteger, BOXContentSDKCacheError) {
+    BOXContentSDKSCacheErrorArchiveFailed = 40000,
```

Comment:
overlapping with BOXContentSDKAppUserError.

Also, do we need one for 'SDKCacheErrorUnknown'?

Also, should the name be more specific to the networking/session/whatever cache for clarity?